doc-content/enterprise-only/openshift/hacep-server-proc.adoc

```diff
@@ -56,7 +84,8 @@ oc get is/openshift-kie-springboot -o template --template='{{range .status.tags}
 +
 . Open the `kubernetes/deployment.yaml` file in a text editor.
 . Replace the existing image URL with the result of the previous command.
-. Remove all characters at the end f the line starting with the `@` symbol, then add `:latest` to  the line. For example: 
+. Replace `<id_user>` with the user ID that you entered in the Docker file in a previous step.
+. Remove all characters at the end of the line starting with the `@` symbol, then add `:latest` to  the line. For example: 
```

Comment:
. Replace `<id_user>` with the user ID that you entered in the Docker file.
. Remove all characters at the end of the line starting with the `@` symbol, then add `:latest` to  the line. For example: